rustler_tests/test/benchmarks.exs

```diff
@@ -0,0 +1,112 @@
+encode_jobs = %{
```

Comment:
Probably not for now, but maybe later: We should move the benchmarks to `rustler_benchmarks`.